contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -442,6 +483,29 @@ TdsProtocolInit(void)
 	TdsRequestCtrl->status = 0;
 
 	MemoryContextSwitchTo(oldContext);
+
+	/* Define the signal handler for SIGIO. */
+	pqsignal(SIGIO, attention_listener_handler);
+
+	/* Assign the parent process to the socket. */
+	if (fcntl(MyProcPort->sock, F_SETOWN, getpid()) < 0) 
+	{
+		/* Todo: Should be FATAL. */
+		elog(LOG, "setting parent process to socket is failed");
+	}
+
+	/* Allow asynchronous notification of pending requests. */
+	if (fcntl(MyProcPort->sock, F_SETFL, FASYNC) < 0) 
+	{
+		/* Todo: Should be FATAL. */
+		elog(LOG, "Settiing ASYNC option to socket failed");
```

Comment:
Error messages (messages, not details) should be sentence fragments, which means they should not start with an upper-case letter.